openapi.yaml

```diff
@@ -469,1250 +491,1393 @@ paths:
-            type: number
-            format: float
-            description: Hours/units defaulted on timesheets for earnCode.
-            example: 173.33
-          shift:
-            type: string
-            description: Shift code used as a default on timesheet for earnCode.
-          code:
-            type: string
-            description: Default earnings code attached to the job and used on timesheet.
-            example: REG
-      useTemporarySsnInd:
-        type: boolean
-        description: When creating or updating a job, this field indicates the employee needs a temporary social security number assigned.
-      employeeInformationReleaseInd:
-        type: boolean
-        description: Indicates the employee has consented to having their employment information shared.
-      salaryInformationReleaseInd:
-        type: boolean
-        description: Indicates the employee accepted or denied to have their salary information shared.
-      salaryInformationReleaseDate:
+        enum:
+          &addressTypeEnums [PA, RN, RB, RE, RK, VO, VP, XX, CM, DM, EO, RH, PR, MA, SV, JA]
+    phoneType:
+      name: filter[phoneType]
+      in: query
+      required: false
+      description: |
+        Filter by phone type. Phone types and their descriptions:
+        - `PA` - Student Alternate Contact
+        - `RN` - Geographic Origin-First Admit
+        - `RB` - Research Billing
+        - `MG` - Mailing Address (Grades)
+        - `RE` - Billing Phone
+        - `RK` - Financial Aid Address
+        - `VO` - Vendor Order
+        - `VP` - Vendor Payment
+        - `VF` - Vendor Fax
+        - `CM` - Current
+        - `EO` - Employee Office (Directory)
+        - `RH` - Residence Hall
+        - `EF` - Employee Office Fax
+        - `MP` - Mobile Phone
+      schema:
         type: string
-        format: date
-        description: The date that the employee accepted or denied to have their salary information shared.
-        example: '1970-01-01'
-      retirement:
-        type: object
-        properties:
-          code:
-            type: string
-            description: Retirement code
-      i9Form:
-        type: object
-        properties:
-          code:
-            type: string
-            description: |
-              Code to indicate the status of the employee's I9 form.
-              - `R` - Received
-              - `N` - Not Received
-              - `T` - Temporary
-              - `E` - Exempt
-      laborDistribution:
-        type: array
-        items:
+        enum:
+          &phoneTypeEnums [PA, RN, RB, MG, RE, RK, VO, VP, VF, CM, EO, RH, EF, MP]
+  responses:
+    '400':
+      description: Bad request
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResult'
+    '404':
+      description: Resource not found
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResult'
+    '409':
```

Comment:
Looks like @ianbrown9475 mentioned this in the last PR: https://github.com/osu-mist/persons-api/pull/85#discussion_r384617963

Thoughts?

Reply:
I missed that. Its in now.